src/main/java/org/jabref/model/jabmap/MindMap.java

```diff
@@ -0,0 +1,49 @@
+package org.jabref.model.jabmap;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class is a data model for a complete mind map
+ */
+public class MindMap {
+
+    private List<MindMapNode> nodes;
+    private List<MindMapEdge> edges;
+
+    public MindMap() {
+        nodes = new ArrayList<>();
+        edges = new ArrayList<>();
+    }
+
+    public MindMap(MindMapNode node) {
+        nodes = new ArrayList<>();
```

Comment:
For readability, I would move all field initializers to the constructor.

Better: Write a builder

You could try to use Lombok's builder: https://projectlombok.org/features/Builder (currently not in use in JabRef. Not sure, whether the other maintainers will accept it)